boards/arduino-uno/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# define the cpu used by the arduino uno board
+export CPU = atmega328p
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB0
```

Comment:
I suggest what Martine said about an "arduino-common" pseudo platform, since lots of variants of arduino-compatible boards exists, sharing the same CPU and other configs. This way we can after separate the specific things in the corresponding Makefile of each device (USB port, programmer, etc.)